Modix.Bot/Modules/UserInfoModule.cs

```diff
@@ -62,8 +62,19 @@ await ReplyAsync("", embed: new EmbedBuilder()
             builder.AppendLine("ID: " + userInfo.Id);
             builder.AppendLine("Profile: " + MentionUtils.MentionUser(userInfo.Id));
 
-            if (userInfo.Status is UserStatus status)
-                builder.AppendLine("Status: " + status.Humanize());
+            if (userInfo.IsBanned)
+            {
+                builder.AppendLine("Status: Banned");
```

Comment:
I think this should be highlighted somehow - my eyes skipped right over it in the screenshot. Something like making it bold or adding a hammer emoji would help to make it stand out.